virtool/startup.py

```diff
@@ -249,7 +255,7 @@ async def startup_b2c(app: App):
     """
     config = get_config_from_app(app)
 
-    if not config.use_b2c:
+    if not (config.use_b2c if hasattr(config, "use_b2c") else False):
```

Comment:
Was accessing it directly causing problems? This attribute should always be set for any config which calls this startup step

Reply:
Some processes in tilt complained app["config"] as no attribute "use_b2c"